app/models/order_item.rb

```diff
@@ -1,4 +1,6 @@
 class OrderItem < ApplicationRecord
   belongs_to :order
   belongs_to :item
+  validates_presence_of :quantity_price
```

Comment:
What's this? Not in the schema they gave us.

Reply:
Resolved on branch (changed name of "price" to "price_per_item" (instead of "quantity_price")